src/Constraint/AbstractConstraint.php

```diff
@@ -1,65 +0,0 @@
-<?php
-
-/*
- * This file is part of composer/semver.
- *
- * (c) Composer <https://github.com/composer>
- *
- * For the full copyright and license information, please view
- * the LICENSE file that was distributed with this source code.
- */
-
-namespace Composer\Semver\Constraint;
-
-/**
- * Base constraint class.
- */
-abstract class AbstractConstraint implements ConstraintInterface
```

Comment:
this class should be kept for BC reasons for now, or you need to bump the version to 2.0.0